### ISSUE TITLE
Environment variable support

### ISSUE THREAD

**Issue by eiabea** [emqtt/emq-auth-http#41]
I am trying to use this auth method for my emqtt docker container, but the environment variables are not set correctly

Variables
```
EMQ_AUTH__HTTP__AUTH_REQ: "http://HTTP_SERVICE:5100/auth/mqtt"
EMQ_AUTH__HTTP__ACL_REQ: "http://HTTP_SERVICE:5100/acl/mqtt"
```

Error
```
14:06:40.212 [error] HTTP http://127.0.0.1:8080/mqtt/auth Error: {failed_connect,[{to_address,{"127.0.0.1",8080}},{inet,[inet],econnrefused}]}
```

emqtt version: 2.1.2

**Comment by turtleDeng** [comment]
@eiabea 

```
EMQ_AUTH__HTTP__AUTH__REQ: "http://HTTP_SERVICE:5100/auth/mqtt"
EMQ_AUTH__HTTP__ACL__REQ: "http://HTTP_SERVICE:5100/acl/mqtt"
```

**Comment by eiabea** [comment]
Thank you for the support, but the doku says:
> The environment variables which with EMQ_ prefix are mapped to configuration file, . get replaced by __.

The config file says:
> auth.http.auth_req

So i think the environment varialbe should be:
EMQ_AUTH__HTTP__AUTH_REQ

**Comment by turtleDeng** [comment]
@eiabea The issue has been fixed in v2.3-beta.2

**Comment by turtleDeng** [comment]
@eiabea 
```
EMQ_AUTH__HTTP__AUTH__REQ: "http:\\/\\/HTTP_SERVICE:5100\\/auth\\/mqtt"
EMQ_AUTH__HTTP__ACL__REQ: "http:\\/\\/HTTP_SERVICE:5100\\/acl\\/mqtt"
```
'/' to be escaped
